content/articles/working-with-styled-components-in-react/index.md

```diff
@@ -0,0 +1,464 @@
+
+This section will create a clone for the [disney+](https://www.disneyplus.com/) landing page and add CSS to its components using the styled-components.
+
+First, we need to create an app in React that will contain our landing page.
+
+Within a folder of your choice, open the [Command prompt](https://www.thewindowsclub.com/how-to-open-command-prompt-from-right-click-menu) and type the following command:
+
+```JavaScript
+npx create-react-app disney-landing-page 
+``` 
+This will initialize and create our react app named disney-landing-page by loading and installing all the React dependencies required for our app.
+Once all the dependencies are installed, a development environment for our React application will be ready.
+
+To get into our project folder, use the command below in the command prompt or the terminal of your code editor:
+
+```bash
+cd disney-landing-page
+
+```
+#### Installing styled-components
+Next, we install the styled-components and the react-router-dom libraries into our project through the commands below:
+
+```bash
+yarn add styled-components
+```
+
+```bash
+yarn add react-router-dom
+```
+#### Starting the development server
+To start the development server for the app, run either of the two commands on your terminal, depending on which package manager you are using. 
+
+```JavaScript
+yarn start
+```
+```JavaScript
+npm start
+```
+The expected result on your web browser is as shown:
+
+
+![Server](/engineering-education/working-with-styled-components/server start up.png)
+
+With everything set for our project, we can now open the project in a code editor and begin writing some code.
+Your expected app structure should look like this :
+
+![App structure](/engineering-education/working-with-styled-components/structure.png)
+
+
+#### Creating our Component 
+
+First, we need to create a folder to store our components. Within the *src* folder in your project structure, create a folder and name it *components*.
+
+![Components folder](/engineering-education/working-with-styled-components/component.png)
+
+In your newly created components folder, create two files and name one *Landing.js* and the other *Header.js*. Within these two files, we will create our components and style them, as we will see shortly.
+
+Before we start working with the CSS-in-JS(styled-components), open the *Landing.js* file and the following code to create our first component: 
+
+We create the Landing component with the code below:
+
+```JavaScript
+import styled from "styled-components";// the styled components library we installed is imported here!
+// This component will render the landing page contents in container.
+const Landing = (props)=>{ // a functional component
+  return 
+     (<Container>
+         <Content>
+         <Content>
+     </Container>);
+}
+export default Landing;
+```
+
+To create our second component, open the *Header.js* file that is in the components folder and add the following code:
```

Comment:
Open the Header.js file in the components folder. Add the following code to the file.